### PR TITLE
FGMP-1.0.1

### PR DESCRIPTION
**Pull request by hyphaltip**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/fgmp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  url: https://github.com/stajichlab/FGMP/archive/v1.0.1.tar.gz
+  sha256: 18a95eacaa96ea24bfcd5b848355c4b1b6845ea1d1e86c014ffdf8927ecc6385
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - perl >=5.10
```
Comment (bgruening) [inline recipes/fgmp/meta.yaml]: I think everything besides the compiler can be removed in build.

File: recipes/fgmp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+source:
+  url: https://github.com/stajichlab/FGMP/archive/v1.0.1.tar.gz
+  sha256: 18a95eacaa96ea24bfcd5b848355c4b1b6845ea1d1e86c014ffdf8927ecc6385
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (bgruening) [inline recipes/fgmp/meta.yaml]: You are not compiling at all isn't it? Do you really need the compiler?

File: recipes/fgmp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+#    - perl $FGMP/src/fgmp.pl -h
+
+#commands:
+#    - perl $FGMP/src/fgmp.pl -g sample.dna 2> log
+#    - blast ?
+
+about:
+  home: https://github.com/stajichlab/FGMP
+  license: 'MIT'
+  summary: 'FGMP: assessing fungal genome completeness and gene content. bioRxiv 049619'
```
Comment (bgruening) [inline recipes/fgmp/meta.yaml]: you can remove the bioRxiv ID here ...

File: recipes/fgmp/meta.yaml (hunk 2 above)
Comment (hyphaltip) [reply recipes/fgmp/meta.yaml]: That’s correct. Let’s remove

File: recipes/fgmp/meta.yaml (hunk 3 above)
Comment (hyphaltip) [reply recipes/fgmp/meta.yaml]: Ok no citations needed. We can remove.

**Comment by hyphaltip** [comment]
To followup I pushed these suggested removals on the branch so I guess check after tests run and see if it can be merged?

**Comment by hyphaltip** [comment]
we are re-submitting paper so would like this to be available for installation as soon as possible.

**Pull request merged**